lib/roken/roken.h.in

```diff
@@ -1260,4 +1260,20 @@ int ROKEN_LIB_FUNCTION rk_socket(int, int, int);
 #include <socket_wrapper.h>
 #endif
 
+#ifndef LLONG_MAX
+#ifdef LONG_LONG_MAX
+#define LLONG_MAX LONG_LONG_MAX
+#endif
```

Comment:
Should we have a `#else`?  and then what, `#error` or attempt to define the missing constant?

```
#ifndef LLONG_MAX
#ifdef LONG_LONG_MAX
#define LLONG_MAX LONG_LONG_MAX
#else
#define LLONG_MAX ((long long)(unsigned long long)(~0ULL & 1ULL<< (sizeof(long long) - 1)))
#endif
#endif
```

or just

```
#ifndef LLONG_MAX
#ifdef LONG_LONG_MAX
#define LLONG_MAX LONG_LONG_MAX
#else
#error "Missing: LLONG_MAX and LONG_LONG_MAX"
#endif
#endif
```